src/aosm/azext_aosm/_params.py

```diff
@@ -91,6 +91,12 @@ def load_arguments(self: AzCommandsLoader, _):
                 "Requires Docker to be installed locally."
             ),
         )
+        c.argument(
+            "nexus",
+            options_list=["--nexus"],
```

Comment:
Not sure I love this. I think adding `vnf-nexus` to the list of definition types would be better. We'd also need to explain why CNF doesn't have an equivalent.